### ISSUE TITLE
java.lang.StringIndexOutOfBoundsException in java.lang.String.startEndAndLength

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jareens/softenido#47]
```
java.lang.RuntimeException: Unable to start activity 
ComponentInfo{com.softenido.audible/com.softenido.audible.AudibleMain}: 
java.lang.StringIndexOutOfBoundsException: length=4; regionStart=0; 
regionLength=9
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1963)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1988)
at android.app.ActivityThread.access$600(ActivityThread.java:125)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1154)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:4431)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:784)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:551)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.StringIndexOutOfBoundsException: length=4; regionStart=0; 
regionLength=9
at java.lang.String.startEndAndLength(String.java:593)
at java.lang.String.substring(String.java:1474)
at com.softenido.audible.AudibleMain.isTitleRepeated(AudibleMain.java:990)
at com.softenido.audible.AudibleMain.preparePlayer(AudibleMain.java:587)
at com.softenido.audible.AudibleMain.onCreate(AudibleMain.java:284)
at android.app.Activity.performCreate(Activity.java:4465)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1049)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1927)
... 11 more
```

Original issue reported on code.google.com by `flik...@gmail.com` on 26 Feb 2013 at 9:11

**Comment by GoogleCodeExporter** [comment]
```
retraced:

java.lang.RuntimeException: Unable to start activity 
ComponentInfo{com.softenido.audible/com.softenido.audible.AudibleMain}: 
java.lang.StringIndexOutOfBoundsException: length=4; regionStart=0; 
regionLength=9
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1963)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1988)
at android.app.ActivityThread.access$600(ActivityThread.java:125)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1154)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:4431)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:784)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:551)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.StringIndexOutOfBoundsException: length=4; regionStart=0; 
regionLength=9
at java.lang.String.startEndAndLength(String.java:593)
at java.lang.String.substring(String.java:1474)
at com.softenido.audible.AudibleMain.boolean 
isTitleRepeated(java.lang.String,java.lang.String)(AudibleMain.java:990)
at com.softenido.audible.AudibleMain.void preparePlayer()(AudibleMain.java:587)
at com.softenido.audible.AudibleMain.void 
onCreate(android.os.Bundle)(AudibleMain.java:284)
at android.app.Activity.performCreate(Activity.java:4465)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1049)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1927)
... 11 more
```

Original comment by `flik...@gmail.com` on 26 Feb 2013 at 9:16

**Comment by GoogleCodeExporter** [comment]
```
fixed with e65da28
```

Original comment by `flik...@gmail.com` on 26 Feb 2013 at 10:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `flik...@gmail.com` on 26 Feb 2013 at 10:48
- Changed state: **Fixed**
